.gitignore

```diff
@@ -25,10 +25,9 @@ cmd/security-file-token-provider/security-file-token-provider
 cmd/security-secretstore-setup/security-secretstore-setup
 cmd/support-logging/support-logging
 cmd/support-notifications/support-notifications
-cmd/support-cron-scheduler/support-cron-scheduler
+cmd/support-scheduler/support-scheduler
 cmd/secrets-config/secrets-config
 cmd/security-bootstrapper/security-bootstrapper
-cmd/security-proxy-auth/security-proxy-auth
```

Comment:
why do you remove this line?

Reply:
It's duplicated to https://github.com/edgexfoundry/edgex-go/pull/5003/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R22